src/confluent_kafka/_model/__init__.py

```diff
@@ -60,7 +61,7 @@ class ConsumerGroupTopicPartitions:
         List of topic partitions information.
     """
 
-    def __init__(self, group_id, topic_partitions=None):
+    def __init__(self, group_id: str, topic_partitions: Optional[List['cimpl.TopicPartition']] = None) -> None:
```

Comment:
Why'd this need the cimpl path prefix to work here? Curious to chat about how the cbinding type errors were worked out in development